site/docs/v1/tech/identity-providers/index.adoc

```diff
@@ -47,6 +48,13 @@ Find the FusionAuth Identity Providers in the UI by navigating to [breadcrumb]#S
 
 image::identity-providers/identity-providers.png[Identity Providers,width=1200]
 
+== Identity Providers and Tenants
+
+Identity providers can be configured to set a limit on the number of links that may be established on a per tenant basis.
```

Comment:
This sort of sounds like the config is limiting the number of links for a user in a tenant. But it is limiting the number of links to a user for a particular IdP, and the configuration is tenant scoped. We should probably re-word this a bit to make it clearer.

Reply:
Re-worded, let me know if I should say something additional about tenant scope.  I keep thinking that the root object field says that but I could double down here too if need be.